client/control/sass/control.scss

```diff
@@ -260,3 +269,12 @@
     font-weight: 800;
   }
 }
+
+/* Buttons */
+.button-icon {
+  background: none;
+  border: none;
+  cursor: pointer;
+  padding: 0;
+  transform: translate(0, -1px);
```

Comment:
Though... Now that I look again, it looks very different than I remember, so I'm just going to remove this altogether. I think I had something weird in my environment previously.